redbot/core/i18n.py

```diff
@@ -159,16 +245,32 @@ def __call__(self, untranslated: str) -> str:
         This will look for the string in the translator's :code:`.pot` file,
         with respect to the current locale.
         """
+        locale = get_locale()
         try:
-            return self.translations[untranslated]
+            return self.translations[locale][untranslated]
         except KeyError:
             return untranslated
 
+    def check_cache_for_locale(self, locale: str) -> bool:
+        """Check is an locale exists with in the cache"""
+        if locale in self.translations:
+            return True
+        return False
+
     def load_translations(self):
         """
         Loads the current translations.
         """
-        self.translations = {}
+        locale = get_locale()
+
+        if locale.lower() == "en-us":
+            # Red is written in en-US, no point in loading it
+            return
+        if locale in self.translations:
+            # Locales cannot be loaded twice as they have an entry in
+            # self.translations, which is created in L#92.
```

Comment:
This line number reference doesn't make sense anymore and even if you update it, it will probably not make sense after some time again. Can you just refer to the relevant method name?